DEVELOPMENT.md

```diff
@@ -0,0 +1,53 @@
+# GNURadio 4.0 Development Environment
+
+## Getting the source
+
+Get the source from the GitHub Repository:
+```bash
+git clone git@github.com:fair-acc/gnuradio4.git
+```
+
+## Building
+### Docker CLI
+To just compile GNURadio4 without installing any dependencies you can just use the Docker image which is also used by our CI builds. The snippet below uses `docker run` to start the container with the current directory mapped into the container with the correct user and group IDs.
+It then compiles the project and runs the testsuite.
+Note that while the binaries inside of `./build` can be accessed on the host system, they are linked against the libraries of the container and will most probably not run on the host system.
+
+```bash
+me@host$ cd gnuradio4
+me@host$ docker run \
+    --user `id -u`:`id -g` \
+    --volume="/etc/group:/etc/group:ro" \
+    --volume="/etc/passwd:/etc/passwd:ro" \
+    --volume="/etc/shadow:/etc/shadow:ro" \
+    --workdir=/build --volume `pwd`:/build  -it \
```

Comment:
maybe rather name the source directory `/src` instead of `/build`, to avoid confusion

Reply:
`/work/src`